data/configuration.yml

```diff
@@ -65,8 +65,14 @@ blacklist:
     path: https://raw.githubusercontent.com/dnswarden/blocklist/master/blacklist-formats/hosts
   - format: domains
     path: https://raw.githubusercontent.com/nextdns/metadata/master/parentalcontrol/bypass-methods
+  - format: hosts
+    path: https://raw.githubusercontent.com/ejrv/VPNs/master/vpn-ipv4.txt
+  - format: hosts
+    path: https://raw.githubusercontent.com/ejrv/VPNs/master/vpn-ipv6.txt
   - format: domains
     path: https://raw.githubusercontent.com/GeorgeForse/VPN-List/master/list.txt
+  - format: domains
+    path: https://dblw.oisd.nl/
```

Comment:
Support for the wildcard domains is a bit finicky at the moment. I have to fix some logic and validation scripts first.
For now, please replace this with the non-wildcard version that you mentioned.

We can replace with wildcard version once we are more confident with the support.